Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/shotgun.yml

```diff
@@ -0,0 +1,9 @@
+- type: entity
+  id: Pellet410ShotgunSpread
+  noSpawn: true
```

Comment:
Needs to be replaced with  `categories: [ HideSpawnMenu ]`

Reply:
done